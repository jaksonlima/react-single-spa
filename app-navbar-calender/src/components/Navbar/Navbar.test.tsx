import { render } from "@testing-library/react";

import { Navbar } from ".";

describe("ResponsiveDrawer", () => {
  it("should render ResponsiveDrawer correctly", () => {
    const { asFragment } = render(<Navbar />);

    expect(asFragment()).toMatchSnapshot();
  });
});
