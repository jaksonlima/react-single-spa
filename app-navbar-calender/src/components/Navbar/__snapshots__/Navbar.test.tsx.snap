// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResponsiveDrawer should render ResponsiveDrawer correctly 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-1s30ybv"
  >
    <div
      class="MuiBox-root css-ncqudi"
    >
      <label
        class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-78trlr-MuiButtonBase-root-MuiIconButton-root"
        role="button"
        tabindex="0"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root MuiSvgIcon-colorPrimary MuiSvgIcon-fontSizeLarge css-1t9wgl6-MuiSvgIcon-root"
          data-testid="KeyboardArrowRightIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M8.59 16.59 13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
          />
        </svg>
        <span
          class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
        />
      </label>
    </div>
    <div
      class="MuiBox-root css-0"
    >
      <div
        class="MuiGrid-root MuiGrid-container css-mft534-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-item css-1f435ry-MuiGrid-root"
        >
          <div
            class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
          >
            <img
              alt="Dasa"
              height="20"
              src="/dasa-logo-1.svg"
              width="32"
            />
          </div>
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-wrap-xs-nowrap css-1y6fhq1-MuiGrid-root"
          >
            <div
              class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
            >
              <label
                class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-78trlr-MuiButtonBase-root-MuiIconButton-root"
                role="button"
                tabindex="0"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall css-17u0i8v-MuiSvgIcon-root"
                  data-testid="AssignmentIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 3h-4.18C14.4 1.84 13.3 1 12 1c-1.3 0-2.4.84-2.82 2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zm-7 0c.55 0 1 .45 1 1s-.45 1-1 1-1-.45-1-1 .45-1 1-1zm2 14H7v-2h7v2zm3-4H7v-2h10v2zm0-4H7V7h10v2z"
                  />
                </svg>
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </label>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item css-1u25xf3-MuiGrid-root"
            >
              <p
                class="MuiTypography-root MuiTypography-body1 MuiTypography-noWrap css-76ijrb-MuiTypography-root"
              >
                Carros
              </p>
            </div>
          </div>
          <hr
            class="MuiDivider-root MuiDivider-fullWidth MuiDivider-flexItem css-1tt9ccx-MuiDivider-root"
          />
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-wrap-xs-nowrap css-18tt7fl-MuiGrid-root"
          >
            <div
              class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
            >
              <label
                class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-78trlr-MuiButtonBase-root-MuiIconButton-root"
                role="button"
                tabindex="0"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall css-17u0i8v-MuiSvgIcon-root"
                  data-testid="EventNoteIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M17 10H7v2h10v-2zm2-7h-1V1h-2v2H8V1H6v2H5c-1.11 0-1.99.9-1.99 2L3 19c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zm0 16H5V8h14v11zm-5-5H7v2h7v-2z"
                  />
                </svg>
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </label>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item css-1u25xf3-MuiGrid-root"
            >
              <p
                class="MuiTypography-root MuiTypography-body1 MuiTypography-noWrap css-76ijrb-MuiTypography-root"
              >
                Criar carro
              </p>
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-wrap-xs-nowrap css-18tt7fl-MuiGrid-root"
          >
            <div
              class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
            >
              <label
                class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-78trlr-MuiButtonBase-root-MuiIconButton-root"
                role="button"
                tabindex="0"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall css-17u0i8v-MuiSvgIcon-root"
                  data-testid="MoreTimeIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M10 8v6l4.7 2.9.8-1.2-4-2.4V8z"
                  />
                  <path
                    d="M17.92 12c.05.33.08.66.08 1 0 3.9-3.1 7-7 7s-7-3.1-7-7 3.1-7 7-7c.7 0 1.37.1 2 .29V4.23c-.64-.15-1.31-.23-2-.23-5 0-9 4-9 9s4 9 9 9 9-4 9-9c0-.34-.02-.67-.06-1h-2.02z"
                  />
                  <path
                    d="M20 5V2h-2v3h-3v2h3v3h2V7h3V5z"
                  />
                </svg>
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </label>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item css-1u25xf3-MuiGrid-root"
            >
              <p
                class="MuiTypography-root MuiTypography-body1 MuiTypography-noWrap css-76ijrb-MuiTypography-root"
              >
                Criar horário
              </p>
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-wrap-xs-nowrap css-18tt7fl-MuiGrid-root"
          >
            <div
              class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
            >
              <label
                class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-78trlr-MuiButtonBase-root-MuiIconButton-root"
                role="button"
                tabindex="0"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall css-17u0i8v-MuiSvgIcon-root"
                  data-testid="InsertInvitationIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M17 12h-5v5h5v-5zM16 1v2H8V1H6v2H5c-1.11 0-1.99.9-1.99 2L3 19c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2h-1V1h-2zm3 18H5V8h14v11z"
                  />
                </svg>
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </label>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item css-1u25xf3-MuiGrid-root"
            >
              <p
                class="MuiTypography-root MuiTypography-body1 MuiTypography-noWrap css-76ijrb-MuiTypography-root"
              >
                Criar calendário
              </p>
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-wrap-xs-nowrap css-18tt7fl-MuiGrid-root"
          >
            <div
              class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
            >
              <label
                class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-78trlr-MuiButtonBase-root-MuiIconButton-root"
                role="button"
                tabindex="0"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall css-17u0i8v-MuiSvgIcon-root"
                  data-testid="RuleIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M16.54 11 13 7.46l1.41-1.41 2.12 2.12 4.24-4.24 1.41 1.41L16.54 11zM11 7H2v2h9V7zm10 6.41L19.59 12 17 14.59 14.41 12 13 13.41 15.59 16 13 18.59 14.41 20 17 17.41 19.59 20 21 18.59 18.41 16 21 13.41zM11 15H2v2h9v-2z"
                  />
                </svg>
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </label>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item css-1u25xf3-MuiGrid-root"
            >
              <p
                class="MuiTypography-root MuiTypography-body1 MuiTypography-noWrap css-76ijrb-MuiTypography-root"
              >
                Criar regras
              </p>
            </div>
          </div>
          <hr
            class="MuiDivider-root MuiDivider-fullWidth MuiDivider-flexItem css-1tt9ccx-MuiDivider-root"
          />
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-wrap-xs-nowrap css-18tt7fl-MuiGrid-root"
          >
            <div
              class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
            >
              <label
                class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-78trlr-MuiButtonBase-root-MuiIconButton-root"
                role="button"
                tabindex="0"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall css-17u0i8v-MuiSvgIcon-root"
                  data-testid="SummarizeIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M15 3H5c-1.1 0-1.99.9-1.99 2L3 19c0 1.1.89 2 1.99 2H19c1.1 0 2-.9 2-2V9l-6-6zM8 17c-.55 0-1-.45-1-1s.45-1 1-1 1 .45 1 1-.45 1-1 1zm0-4c-.55 0-1-.45-1-1s.45-1 1-1 1 .45 1 1-.45 1-1 1zm0-4c-.55 0-1-.45-1-1s.45-1 1-1 1 .45 1 1-.45 1-1 1zm6 1V4.5l5.5 5.5H14z"
                  />
                </svg>
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </label>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item css-1u25xf3-MuiGrid-root"
            >
              <p
                class="MuiTypography-root MuiTypography-body1 MuiTypography-noWrap css-76ijrb-MuiTypography-root"
              >
                Relatórios
              </p>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-item css-1f435ry-MuiGrid-root"
        >
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-wrap-xs-nowrap css-18tt7fl-MuiGrid-root"
          >
            <div
              class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
            >
              <label
                class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-78trlr-MuiButtonBase-root-MuiIconButton-root"
                role="button"
                tabindex="0"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall css-17u0i8v-MuiSvgIcon-root"
                  data-testid="HomeWorkIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M1 11v10h5v-6h4v6h5V11L8 6z"
                  />
                  <path
                    d="M10 3v1.97l7 5V11h2v2h-2v2h2v2h-2v4h6V3H10zm9 6h-2V7h2v2z"
                  />
                </svg>
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </label>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item css-1u25xf3-MuiGrid-root"
            >
              <p
                class="MuiTypography-root MuiTypography-body1 MuiTypography-noWrap css-16ran6o-MuiTypography-root"
              >
                SELECIONAR
              </p>
            </div>
          </div>
          <hr
            class="MuiDivider-root MuiDivider-fullWidth MuiDivider-flexItem css-1tt9ccx-MuiDivider-root"
          />
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-wrap-xs-nowrap css-18tt7fl-MuiGrid-root"
          >
            <div
              class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
            >
              <div
                class="MuiBox-root css-1jhfqhr"
              >
                <strong
                  class="MuiTypography-root MuiTypography-body1 css-ndaqpv-MuiTypography-root"
                >
                  JL
                </strong>
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item css-1u25xf3-MuiGrid-root"
            >
              <p
                class="MuiTypography-root MuiTypography-body1 MuiTypography-noWrap css-1cvrmz3-MuiTypography-root"
              >
                Jakson Lima
              </p>
              <p
                class="MuiTypography-root MuiTypography-body1 MuiTypography-noWrap css-1q9y42x-MuiTypography-root"
              >
                f10506505901
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
